public/components/datasources/components/manage/accelerations/acceleration_details_flyout.tsx

```diff
@@ -101,18 +123,28 @@ export const AccelerationDetailsFlyout = (props: AccelerationDetailsFlyoutProps)
       });
   };
 
+  useEffect(() => {
+    if (operationSuccess !== undefined) {
+      if (operationSuccess) {
```

Comment:
Won't both of these if statements result the exact same way? (operationSuccess) is the same as resulting to 'if operationSuccess is defined'

Reply:
good catch. just clear it up.